storage/database/src/env.rs

```diff
@@ -81,13 +80,13 @@ pub trait Env: Sized {
     /// Open the database environment, using the passed [`Config`].
     ///
     /// # Invariants
-    /// This function **must** create all tables listed in [`crate::tables`].
+    /// This function does not create any tables.
     ///
-    /// The rest of the functions depend on the fact
-    /// they already exist, or else they will panic.
+    /// You must create all possible tables with [`EnvInner::create_db`]
+    /// before attempting to open any.
```

Comment:
## `cuprate_database::Env` 1/3

`cuprate_database` no longer has any idea what the user is doing since it is _only_ a database abstraction. This means it doesn't know what tables the user wants, so this invariant is passed on to users now.

This means after `cuprate_database::Env::open`, you have a database but with no tables and it is up to you to create them (where as before `cuprate_database` created all the blockchain tables).